### DISCUSSION TITLE
[Feature] Custom pre/post updating script

### DISCUSSION THREAD

**Discussion by WindyNova** [ScoopInstaller/Scoop#5737] [Ideas]
Is it possible to add a feature that allows the execution of a specific script both before and after the updating process? Since you cant update scoop apps while they are running, i need it to terminate the running process and bring back the app after the upgrade is complete.

**Comment by rashil2000** [comment]
You can use the `pre_uninstall` field in the manifest to kill the program. For more info, see https://github.com/ScoopInstaller/Scoop/discussions/5648
